crates/rolldown_common/src/inner_bundler_options/mod.rs

```diff
@@ -36,6 +36,7 @@ pub struct BundlerOptions {
   pub platform: Option<Platform>,
   pub shim_missing_exports: Option<bool>,
   // --- options for output
+  pub name: Option<String>,
```

Comment:
```suggestion
  pub iife_name: Option<String>,
```

`name` is kind of confusing here. We can't change it in package `rolldown` due to compatibility with rollup. But let's make it more meaningful in the core side.

Reply:
It also will using at `umd/amd`, so here using `name` maybe is good for me. I prefer js options same as rust options, avoid caused some issue.

Reply:
> I prefer js options same as rust options

Reasonable comes first if there are no compatibility issues.

But I didn't thought of `umd/amd`. Let's keep using `name` before we come up with better name.